Source/CesiumRuntime/Private/GlobeAwareDefaultPawn.cpp

```diff
@@ -129,7 +129,9 @@ void AGlobeAwareDefaultPawn::FlyToLocationECEF(
 
   // The source and destination rotations are expressed in East-South-Up
   // coordinates.
-  this->_flyToSourceRotation = Controller->GetControlRotation().Quaternion();
+  this->_flyToSourceRotation =
```

Comment:
Originally, I tried to run the new test in Edit mode, but it crashed because there was no Controller. So I added this check. But then there were other problems running it in Edit mode (mostly that the flights are completely disabled if there's no game world), so I gave up and made the test use Play-in-Editor mode. I left this change in, though, because not crashing is better than crashing. I believe it's possible to have no Controller even in a game, if the pawn isn't posessed.

Really, camera flights should have nothing to do with Controllers or even Pawns. It should simply be a component that we can add to any Actor, similar to what we've done in Unity.

But I'm not inclined to get into any of that as part of this PR.

Reply:
I cleaned this up a little based on your suggestion. It now aborts earlier when there's no Controller, and logs an error message.

Reply:
Looks great, thanks.